deploy/kube-config/standalone-test/heapster-summary-controller.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
```

Comment:
It still defaults to heapster.controller.yaml, but it's configurable with a [flag](https://github.com/kubernetes/heapster/blob/heapster-scalability/integration/heapster_api_test.go#L44)